bin/create-digests.py

```diff
@@ -0,0 +1,114 @@
+import os
+import json
+import hashlib
+import logging
+
+logging.basicConfig(encoding='utf-8', level=logging.INFO)
+
+def round_floats(data):
+    if type(data) == list:
+        for i in range(len(data)):
+            round_floats(data[i])
+    else:
+        for key in data:
+            the_type = type(data[key])
+            if the_type == dict:
+                round_floats(data[key])
+            elif the_type == list:
+                round_floats(data[key])
+            elif the_type == float:
+                data[key] = round(data[key],2)
+
+def sort_arrays(data):
+    if type(data) == list:
+        if len(data) > 0:
+            if type(data[0]) == dict:
+                data.sort(key=lambda x: tuple([str(x[key]) for key in x.keys()]))
+            else:
+                data.sort()
+    else:
+        for key in data:
+            the_type = type(data[key])
+            if the_type == dict:
+                sort_arrays(data[key])
+            elif the_type == list:
+                sort_arrays(data[key])
+
+def redact(data):
+    if type(data) == dict:
+        if 'date_processed' in data:
+            data['date_processed'] = '***'
+        else:
+            for key in data.keys():
+                if key.startswith('top_') :
+                    # Redact names for items with duplicate amounts and last item in case the next
+                    # was duplicated.  We have to do this now because the ordering for these lists
+                    # are undefined by the amounts are the same
```

Comment:
Don't you mean: "undefined **if** the amounts are the same"?

Reply:
yes - mistyped.  I'll fix